builton_sdk/api_models/component.py

```diff
@@ -71,13 +99,28 @@ def simple_query(self,
                      body=None,
                      api_path=None,
                      res_constructor=None,
-                     json=False):
+                     json=False,
+                     **kwargs):
 
         # TODO: add pagination
         # TODO: add object expand
 
+        if url_params is None:
+            url_params = {}
+
         resource = self.build_resource(_id, resource, api_path)
 
+        if _type == 'get':
+            for param in ALLOWED_URL_PARAMS:
```

Comment:
We can wrap this `for` into a method, it seems to be used couple time

Reply:
Reused the `_build_url_params` method here.